_news/2019-09-17-deprecate-python-installer.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Terraform: Official Installation Path of Forseti Security
+author: Hannah Shin
+---
+
+Forseti Community,
+
+We’ve heard from many of our users that Forseti’s Deployment Manager-based 
```

Comment:
Let's avoid the negativity for DM, how about something like this?
```
We’ve heard from many of our users that we should focus on delivering
a faster, simpler, and more consistent deployment experience with Terraform. 
```

Reply:
How about:

```We’ve heard feedback from many of our users around the current Forseti installation and upgrade path, and we are focusing on delivering a faster, simpler, and more consistent experience with Terraform. ```